### ISSUE TITLE
testExpireAtWithLong fails on 32bit (i686) build

### ISSUE THREAD

**Issue by carlwgeorge** [phpredis/phpredis#838]
phpredis 3.0.0
PHP 7.0.7 (from [IUS](https://ius.io))
Redis 2.8.24/3.0.7/3.2.0 (from [IUS](https://ius.io))
CentOS 6 32bit (i686)

With the above environment, testExpireAtWithLong fails.  It does not fail on x86_64 builds.

```
testExpireAtWithLong
Warning: Redis::setex() expects parameter 2 to be integer, float given in /builddir/build/BUILD/php70u-pecl-redis-3.0.0/NTS/tests/RedisTest.php on line 461
[FAILED]
```

I'm an RPM packager, not a PHP developer, but I'll help to diagnose this any way I can.

**Comment by carlwgeorge** [comment]
@michael-grunder @yatsukhnenko ping?

**Comment by yatsukhnenko** [comment]
@carlwgeorge did you try code from `develop` branch?

**Comment by carlwgeorge** [comment]
I didn't try this with the develop branch yet, but I did just try with the 3.1.0 pecl release.  This test still fails, along with five more.  They are the same failures reported in #1056.

**Comment by yatsukhnenko** [comment]
Fixed in PR #1075
